### ISSUE TITLE
iOS: zooming and moving available only after a longer hold on the timeline

### ISSUE THREAD

**Issue by almute** [almende/vis#2432]
I'm having a problem with moving and zooming the timeline on iOS devices (iOS 10).
The function is available only after a longer hold with the finger on the timeline.
Is there a possibility to change this behaviour or is this a bug?

Thanks, Almute

**Comment by Tooa** [comment]
Maybe a duplicate of https://github.com/almende/vis/issues/2123?

**Comment by mojoaxel** [comment]
@almute Please make sure that you did not activate the option `clickToUse`.

**Comment by almute** [comment]
clickToUse is not activated.
And I don't think that it is a duplicate of #2123  because after longer holding on the timeline it works fine until the timeline is refreshed.
